Content.Server/StationRecords/Systems/StationRecordsSystem.cs

```diff
@@ -54,6 +57,28 @@ private void OnPlayerSpawn(PlayerSpawnCompleteEvent args)
             return;
 
         CreateGeneralRecord(args.Station, args.Mob, args.Profile, args.JobId, stationRecords);
+
+        /*var query = EntityQueryEnumerator<SectorStationRecordComponent>();
+
+        while (query.MoveNext(out var stationGridUid, out var comp))
+        {
+            if (TryComp<StationMemberComponent>(stationGridUid, out var stationMemberComponent))
+            {
+                var stationEntityUid = stationMemberComponent.Station;
+
+                CreateGeneralRecord(stationEntityUid, args.Mob, args.Profile, args.JobId, stationRecords);
+
+                /*
+                if (TryComp<StationRecordKeyStorageComponent>(targetId, out var keyStorage)
+                && stationEntityUid != null
+                && keyStorage.Key != null)
+                {
+                    if (!TryGetRecord<GeneralStationRecord>(Key.Value, out var record))
+                        continue;
+            }
+            }
+
+        }*/
```

Comment:
?

Reply:
With this it was original block present. 
And existed as commented to assist with upstream merges and merge conflicts that could happen.

As changed decently below.
Though if this part doesn't need to stay then can remove it